oonidata/models/observations.py

```diff
@@ -378,3 +378,34 @@ class HTTPMiddleboxObservation(MeasurementMeta):
     hfm_diff: Optional[str] = None
     hfm_failure: Optional[str] = None
     hfm_success: Optional[bool] = None
+
+
+@add_slots
+@dataclass
+class CircumventionToolObservation(MeasurementMeta):
```

Comment:
While this looks really good, I think we should invest a bit of time to define this Observation data model such that's flexible enough for all future circumvention tools.

Ideally it would be something that can adapt nicely to new circumvention tools as we put them out and it's probably worth checking with @ainghazal what his thoughts are on the topic.

Some of the considerations to keep in mind are the following:
* Schema migrations are a pain, so the less we do the better it is
* It's easier to add new columns, than it is to change an existing column
* We should factor in schema evolution in such a way where we make it as future proof as possible, but where we anticipate changes, they are done through the addition of new columns, rather than changes to existing ones